db/migrate/20180806152607_create_taxes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTaxes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :taxes do |t|
+      t.string :date
```

Comment:
as I understand, we need to have a period of dates, when we need to send declaration to a tax inspection. could you please make it a range and name it with `declaration_period` or smth like that.